package.json

```diff
@@ -15,7 +15,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy": "yarn build && aws s3 sync build/ s3://codepurpose.io && aws cloudfront create-invalidation --distribution-id E36PWAVS8UMFGX --paths '/*'"
```

Comment:
@MilioFrankie I added this, which will allow you to deploy to production. We will have to get the keys setup on your machine or move this deploy task to an online solution like circleci which can test our app then deploy it once changes make it to master.

Reply:
Sounds good, thanks for doing this!